modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/url/FDSAPIURLBuilder.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.url;
+
+/**
+ * @author Daniel Sanz
+ */
+public interface FDSAPIURLBuilder {
+
+	public FDSAPIURLBuilder addParameter(String name, String value);
+
+	public FDSAPIURLBuilder addQueryString(String queryString);
```

Comment:
Most often it's called parameters, but this should be acceptable synonym :+1:

Reply:
I've inspired on other similar patterns we have such as the `DynamicServletRequest` or `HttpComponentsUtil` in order to make a better separation in terms of naming